integration/simulation/params/params.go

```diff
@@ -20,4 +26,9 @@ type SimParams struct {
 	L1EfficiencyThreshold     float64
 	L2EfficiencyThreshold     float64
 	L2ToL1EfficiencyThreshold float64
+
+	// TxHandler defines how the simulation should unpack transactions
+	TxHandler rollupcontractlib.TxHandler
+	// ContractAddr defines the contract address
```

Comment:
@otherview Can we clarify this is the rollup contract address?

Reply:
Done 👍 . Also changed to `MgmtContractAddr`